<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ESTO_CAO</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Ressources.qrc">
    <normaloff>:/Images/monitor.png</normaloff>:/Images/monitor.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>565</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouveau"/>
    <addaction name="actionOuvrir"/>
    <addaction name="actionEnregistrer"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="actionA_propos"/>
   </widget>
   <widget class="QMenu" name="menuOutils">
    <property name="title">
     <string>Outils</string>
    </property>
    <addaction name="actionLigne"/>
    <addaction name="actionRectangle"/>
    <addaction name="actionCercle"/>
    <addaction name="actionPolygon"/>
    <addaction name="actionCouleur"/>
    <addaction name="actionRemplissement"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionAnnuler"/>
    <addaction name="actionRefaire"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuOutils"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLigne"/>
   <addaction name="actionRectangle"/>
   <addaction name="actionCercle"/>
   <addaction name="actionPolygon"/>
   <addaction name="actionCouleur"/>
   <addaction name="actionRemplissement"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>137</width>
     <height>137</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>170</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Commands</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOuvrir">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/Images/folder.png</normaloff>:/Images/folder.png</iconset>
   </property>
   <property name="text">
    <string>Ouvrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionEnregistrer">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/Images/floppy-disc.png</normaloff>:/Images/floppy-disc.png</iconset>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/Images/window.png</normaloff>:/Images/window.png</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/Images/book.png</normaloff>:/Images/book.png</iconset>
   </property>
   <property name="text">
    <string>Documentation</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/Images/info.png</normaloff>:/Images/info.png</iconset>
   </property>
   <property name="text">
    <string>A propos..</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/Images/rect.png</normaloff>:/Images/rect.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
   <property name="shortcut">
    <string>Alt+R</string>
   </property>
  </action>
  <action name="actionCercle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/Images/ellipse.png</normaloff>:/Images/ellipse.png</iconset>
   </property>
   <property name="text">
    <string>Cercle</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionPolygon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/Images/polygon.png</normaloff>:/Images/polygon.png</iconset>
   </property>
   <property name="text">
    <string>Polygon</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
  </action>
  <action name="actionLigne">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/Images/ligne.png</normaloff>:/Images/ligne.png</iconset>
   </property>
   <property name="text">
    <string>Ligne</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionNouveau">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/Images/document.png</normaloff>:/Images/document.png</iconset>
   </property>
   <property name="text">
    <string>Nouveau</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAnnuler">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/Images/arrow-left.png</normaloff>:/Images/arrow-left.png</iconset>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRefaire">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/Images/arrow-right.png</normaloff>:/Images/arrow-right.png</iconset>
   </property>
   <property name="text">
    <string>Refaire</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCouleur">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/Images/colors.png</normaloff>:/Images/colors.png</iconset>
   </property>
   <property name="text">
    <string>Couleur</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionRemplissement">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/Images/remplissement.png</normaloff>:/Images/remplissement.png</iconset>
   </property>
   <property name="text">
    <string>Remplissement</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Ressources.qrc"/>
 </resources>
 <connections/>
</ui>
